docs/source/release_notes.rst

```diff
@@ -27,6 +27,7 @@ v1.11.0 Jun 30, 2022
     * Testing Changes
         * Fix pandas warning and reduce dask .apply warnings (:pr:`2145`)
         * Pin graphviz version used in windows tests (:pr:`2159`)
+        * Standardize imports in `primitive_tests` folder (:pr:`2166`) 
```

Comment:
This change looks bigger than just the `primitive_tests` folder, so maybe update the entry accordingly. Also, this needs to move to the Future Release section (which needs to be uncommented) and not in the `v1.11.0` release section.